pkg/aggregator/aggregator.go

```diff
@@ -195,6 +198,19 @@ func init() {
 	tagsetTlm = newTagsetTelemetry([]uint64{90, 100})
 
 	aggregatorExpvars.Set("MetricTags", expvar.Func(expMetricTags))
+
+	expvarcollector.RegisterExpvarReport("aggregator", func() (interface{}, error) {
+		aggregatorStatsJSON := []byte(aggregatorExpvars.String())
+		aggregatorStats := make(map[string]interface{})
+		json.Unmarshal(aggregatorStatsJSON, &aggregatorStats) //nolint:errcheck
+		s, err := checkstats.TranslateEventPlatformEventTypes(aggregatorStats)
+		if err != nil {
+			log.Debugf("failed to translate event platform event types in aggregatorStats: %s", err.Error())
```

Comment:
Logs in init could be problematic since the logging framework may not be fully initialized

Reply:
I had no idea about that constraint. Thank you so much for pointing it out. I will remove it

Reply:
Note that we already log things before being ready, those logs are added in a buffer and flushed when the logger is initialized (but it's still better to avoid that when possible)